reviewing/plugin-commands-outdated/src/outdated.ts

```diff
@@ -312,9 +314,16 @@ function renderOutdatedJSON (outdatedPackages: readonly OutdatedPackage[], opts:
   return JSON.stringify(outdatedPackagesJSON, null, 2)
 }
 
-function sortOutdatedPackages (outdatedPackages: readonly OutdatedPackage[]) {
+function sortOutdatedPackages (outdatedPackages: readonly OutdatedPackage[], opts?: { sortField?: 'name' }) {
+  const sortKey = opts?.sortField
+  let comparators = []
+  if (sortKey === 'name') {
+    comparators = [DEFAULT_COMPARATORS[1]]
```

Comment:
It's better to define a meaningful variable such as `NAME_COMPARATORS`, the index 1 is hard to understand

Reply:
Makes sense, it's been updated.